config/environments/staging.rb

```diff
@@ -68,7 +68,7 @@
   config.action_mailer.raise_delivery_errors = false
   # config.action_mailer.perform_deliveries = true
   config.action_mailer.delivery_method = :letter_opener_web
-  config.action_mailer.smtp_settings = { address: 'mailrelay.blackducksoftware.com',
+  config.action_mailer.smtp_settings = { address: 'mailhost.synopsys.com',
```

Comment:
This should remain as mailrelay.blackducksoftware.com for staging.  Not sure this will be valid, but it shouldn't go through the production mail server.